xl/player/gst/sink.py

```diff
@@ -61,35 +61,65 @@
         "name": "JACK",
         "pipe": "jackaudiosink"
     },
+    "directsoundsink": {
```

Comment:
I think just `directsound` would be better here, to match the other names.

Also, a WASAPI entry would make sense as well.

Reply:
The key is stored in settings, so it would break backwards compatibility to change it.